Remotion/Web/Development.WebTesting.TestSite.Shared/AnchorTest.aspx.cs

```diff
@@ -32,6 +33,11 @@ protected override void OnInit (EventArgs e)
       MyHtmlAnchor.ServerClick += ServerClick;
     }
 
+    protected override CspHeader GetCspHeader ()
+    {
+      return base.GetCspHeader().AddDirectiveValue(CspDirectives.ScriptSrc, "'nonce-testsite'");
+    }
+
```

Comment:
See other comments. There are a few more places so I am going to stop making comments for those.